news/_posts/releases/2022-10-26-prestashop-8-0-0-available.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title:  PrestaShop 8.0 Is Available
+subtitle: The final version is here!
+date: 2022-10-26
+authors: [ PrestaShop ]
+image: /assets/images/2022/10/bannerPS8.png
+icon: icon-leaf
+tags:
+- version
+- major
+- releases
+- 8.0
+- 8.0.x
+---
+
+The maintainers team is pleased to announce that PrestaShop 8.0 is officially available.
+
+![PrestaShop 8.0 is available!](/assets/images/2022/10/bannerPS8.png)
+
+PrestaShop 8, the first major version since the original release of PrestaShop 1.7.0 back in 2016, is an important milestone for the project.
+
+A significant cleanup has been performed in the codebase, with several internal components updated, modified or removed. PrestaShop 8 is also the project’s first company-neutral release, meaning that it no longer includes commercial products and services built-in.
+
+In addition to [over 170 bug fixes](https://github.com/PrestaShop/PrestaShop/issues?page=1&q=is%3Aissue+is%3Aclosed+milestone%3A8.0.0+label%3ABug+-label%3A8.0.x+-label%3APrerelease+-label%3A%22CI%2FCD%22) (including 4 new fixes since the [Release Candidate](https://build.prestashop-project.org/news/prestashop-8-0-rc1-release/)), PrestaShop 8 brings compatibility with PHP 8.1, updated libraries including Symfony 4.4, an improved experimental product page, a new security page, and many other features for developers and merchants alike. You can read all about them here: [What’s new in PrestaShop 8](https://www.prestashop-project.org/releases/prestashop80).
```

Comment:
```suggestion
In addition to [over 170 bug fixes](https://github.com/PrestaShop/PrestaShop/issues?page=1&q=is%3Aissue+is%3Aclosed+milestone%3A8.0.0+label%3ABug+-label%3A8.0.x+-label%3APrerelease+-label%3A%22CI%2FCD%22) (including 4 new fixes since the [release candidate](https://build.prestashop-project.org/news/prestashop-8-0-rc1-release/)), PrestaShop 8 brings compatibility with PHP 8.1, updated libraries including Symfony 4.4, an improved experimental product page, a new security page, and many other features for developers and merchants alike. You can read all about them here: [What’s new in PrestaShop 8](https://www.prestashop-project.org/releases/prestashop80).
```